server/src/main/java/org/opensearch/index/translog/transfer/TransferService.java

```diff
@@ -45,9 +45,12 @@ void uploadBlobAsync(
 
     void deleteBlobs(Iterable<String> path, List<String> fileNames) throws IOException;
 
-    void deleteBlobsAsync(Iterable<String> path, List<String> fileNames, ActionListener<Void> listener);
```

Comment:
Why are we removing these methods? Is it caller's responsibility to call `delete` in async manner? `TransferService` interface becomes inconsistent with this. Upload supports async with `uploadBlobAsync` but for download, it is not supported.

Reply:
Have made the interface consistent. Incorporated.